src/AppContainer.tsx

```diff
@@ -1,29 +1,21 @@
-import { NetworkBar, Overlay } from "@govtechsg/tradetrust-ui-components";
```

Comment:
are u doing the other story in this PR too? the one that is to remove the network bar.

Reply:
Yeap, removing the network bar then move to verify page